src/Fixer/Naming/NoHomoglyphNamesFixer.php

```diff
@@ -233,9 +233,8 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
             }
 
             $replaced = Preg::replaceCallback('/[^[:ascii:]]/u', static function (array $matches) {
-                return isset(self::$replacements[$matches[0]])
-                    ? self::$replacements[$matches[0]]
-                    : $matches[0]
+                return self::$replacements[$matches[0]]
```

Comment:
Maybe we can make this statement single line?

Reply:
done